app/src/widgets/todoist/__tests__/index.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { shallow, ShallowWrapper } from "enzyme";
+
+import { widgetProps } from "common/utils/mock";
+
+import Todoist from "../index";
+
+describe("<Todoist />", () => {
+  let wrapper: ShallowWrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <Todoist
+        {...widgetProps}
+        id="todoist-mock-id"
+        token="xxxx"
+      />
+    );
+  });
+
+  it("renders without error", () => {
+    expect(wrapper.isEmptyRender()).toBe(false);
```

Comment:
FYI - the default (= generated) tests are currently very basic, because I'm planning to replace `enzyme` with `react-testing-library` in the future. If you're experienced with `react-testing-library` (which I'm not that much, yet 😅  ), feel free to write some basic tests. Otherwise you can ignore it for now 😉